jenkins/dockerfiles/dt-validation/Dockerfile

```diff
@@ -9,4 +9,15 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 
 RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 500
 
-RUN pip3 install git+https://github.com/devicetree-org/dt-schema.git@master
+RUN pip3 install --system git+https://github.com/devicetree-org/dt-schema.git@master
+
+RUN sudo apt-get update && install \
```

Comment:
```suggestion
RUN apt-get update && apt-get install \
```

Reply:
No need to run `sudo` as this is already run as `root` in the Docker container, but `apt-get` needs to be run as `install` is not a command on its own.

Reply:
Also `apt-install -y` is needed to run non-interactively and `--no-install-recommends` is to avoid installing a bunch of things not explicitly required in the Dockerfile.